lib/commons/color/get-background-color.js

```diff
@@ -47,59 +48,57 @@ export default function getBackgroundColor(
 }
 
 function _getBackgroundColor(elm, bgElms, shadowOutlineEmMax) {
+  const elmStack = getBackgroundStack(elm);
+  if (!elmStack) {
+    return null;
+  }
+
+  const textRects = getVisibleChildTextRects(elm);
   let bgColors = getTextShadowColors(elm, { minRatio: shadowOutlineEmMax });
   if (bgColors.length) {
     bgColors = [{ color: bgColors.reduce(flattenShadowColors) }];
   }
 
-  const elmStack = getBackgroundStack(elm);
-  const textRects = getVisibleChildTextRects(elm);
-
   // Search the stack until we have an alpha === 1 background
-  (elmStack || []).some(bgElm => {
+  for (let i = 0; i < elmStack.length; i++) {
```

Comment:
By turning this into a loop instead of a `some` we can return early inside the loop rather than trying to return after the `some` has already gone through all elements of the stack.